src/commands/ai/tellme.ts

```diff
@@ -0,0 +1,80 @@
+D'autant que je suis un matheux, que je suis partisan de l'apprentissage poussé de la théorie, c'est d'ailleurs les études que j'ai fait.
+
+Mais il y a une différence entre suivre un cours et apprendre par soi-même.
+
+Et je reste convaincu qu'il est quasiment impossible d'apprendre l'algorithmique par soi-même sans s'appuyer sur un langage pour pouvoir lancer son code et visualiser ce que l'on apprend.
+
+Et même avec un prof, il faut un esprit très matheux pour faire de l'algèbre de bool et de l'algorithmique théorique sans visualiser tout cela à travers l'informatique.
+
+Je ne dis pas que ce n'est pas intéressant, au contraire, j'adore ça, mais ça requiert un esprit bien plus mathématique que ce qui est nécessaire pour faire du web.
+
+En particulier le web est un domaine très spécial. Une bonne partie du web ne s'apparente même pas à de la programmation et de manière générale, l'algorithmique ne représente pas la majorité du travail, même en développement backend.
+Je reste convaincu qu'il faut l'apprendre avant de commencer, ne serait-ce que pour savoir ce que l'on aime faire.
+Mais si l'objectif c'est le web, le chemin théorique me paraît être la bonne façon de se décourager.
+
+À l'inverse, l'apprentissage d'un langage permet de comprendre beaucoup de concepts, qu'il aura tout le loisir d'agrémenter de mathématiques si finalement ce n'est pas tant le web qui l'intéresse dans la programmation.</s>
+[INST] {prompt} [/INST]`;
+
+export const TELLME: Command = {
+    data: new SlashCommandBuilder()
+        .setName('tellme')
+        .setDescription("Demande à Zadier ce qu'il en pense")
+        .addStringOption((option) => option.setName('prompt').setDescription('Le prompt').setRequired(true))
+        .addNumberOption((option) =>
+            option
+                .setName('max_new_tokens')
+                .setDescription('Le nombre maximum de jetons que le modèle doit générer en sortie (1024 par défaut)')
+                .setMinValue(0)
+                .setMaxValue(10000)
+                .setRequired(false),
+        ),
+    execute: async (interaction) => {
+        await interaction.deferReply();
+        const input = {
+            prompt: interaction.options.get('prompt')?.value as string,
+            max_new_tokens: (interaction.options.get('max_new_tokens')?.value as number) || 1024,
+            temperature: 0.6,
+            top_p: 0.9,
+            top_k: 50,
+            presence_penalty: 0,
+            frequency_penalty: 0,
+            prompt_template: PROMPT_TEMPLATE,
+        };
+        let msg = '';
+        let last_time = Date.now();
+        for await (const event of replicate.stream('mistralai/mixtral-8x7b-instruct-v0.1', { input })) {
+            if (event.event === 'output') {
+                msg += event.data;
+            }
+            if (Date.now() - last_time > 500 && msg !== '') {
+                await interaction.editReply(msg);
+                last_time = Date.now();
+            }
+        }
```

Comment:
you use that piece of code in two files, you probably shoud move it to the `replicate.ts` or a utils file